components/brave_wallet/common/eth_request_helper_unittest.cc

```diff
@@ -133,21 +133,21 @@ TEST(EthResponseHelperUnitTest, ShouldCreate1559Tx) {
       "0x7f84E0DfF3ffd0af78770cF86c1b1DdFF99d51CA";
   const std::string hw_address = "0x7f84E0DfF3ffd0af78770cF86c1b1DdFF99d51CC";
 
-  mojom::AccountInfoPtr primary_account =
-      mojom::AccountInfo::New("0x7f84E0DfF3ffd0af78770cF86c1b1DdFF99d51C8",
-                              "primary", false, nullptr, mojom::CoinType::ETH);
+  mojom::AccountInfoPtr primary_account = mojom::AccountInfo::New(
+      "0x7f84E0DfF3ffd0af78770cF86c1b1DdFF99d51C8", "primary", false, nullptr,
+      mojom::CoinType::ETH, mojom::kDefaultKeyringId);
   mojom::AccountInfoPtr ledger_account = mojom::AccountInfo::New(
       ledger_address, "ledger", false,
       mojom::HardwareInfo::New("m/44'/60'/1'/0/0", "Ledger", "123"),
-      mojom::CoinType::ETH);
+      mojom::CoinType::ETH, mojom::kDefaultKeyringId);
   mojom::AccountInfoPtr trezor_account = mojom::AccountInfo::New(
       trezor_address, "trezor", false,
       mojom::HardwareInfo::New("m/44'/60'/1'/0/0", "Trezor", "123"),
-      mojom::CoinType::ETH);
+      mojom::CoinType::ETH, mojom::kDefaultKeyringId);
   mojom::AccountInfoPtr hw_account = mojom::AccountInfo::New(
       hw_address, "hw", false,
       mojom::HardwareInfo::New("m/44'/60'/1'/0/0", "Hardware", "123"),
-      mojom::CoinType::ETH);
+      mojom::CoinType::ETH, mojom::kDefaultKeyringId);
```

Comment:
These should be optional for ETH

Reply:
Actually i think we can keep it non-optional, because every account has unique keyring it is attached

Reply:
Only Filecoin needs to know that info and keyring concept should be opaque to front end. The stuff front end uses to distinguish accounts is CoinType.